src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends Figure {
+    static final double MAX_NUMBER = 100.0;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(TypeOfFigure.values().length);

```

Comment:
```suggestion
        int index = random.nextInt(TypeOfFigure.values().length);

```

you have class variable random, why do u create new object? :)